MUI_Core/Modules/Chat/BlizzardFrames/ChatFrame.lua

```diff
@@ -186,5 +192,17 @@ function C_ChatModule:SetUpBlizzardChatFrame(_, chatFrameID)
 		_G.BNToastFrame.SetPoint = tk.Constants.DUMMY_FUNC;
 	end
 
+  data.ProcessedChatFrames[chatFrameName] = chatFrame;
+
 	return chatFrame;
+end
+
+function C_ChatModule:SetUpAllBlizzardFrames(data)
+  for _, chatFrameName in ipairs(_G.CHAT_FRAMES) do
+    local chatFrame = self:SetUpBlizzardChatFrame(chatFrameName);
+    if (changeGameFont) then
```

Comment:
`changeGameFont` is no longer in scope. Where you moved it from you had the following 2 required lines, but now they are missing:

```lua
  local changeGameFont = db.global.core.changeGameFont;
  local muiFont = tk.Constants.LSM:Fetch("font", db.global.core.font);
```

Maybe you could move this logic into the SetUpBlizzardChatFrame function?

Reply:
Changed this. Also added a check for chatFrame in the `if (changeGameFont) then` to `if (changeGameFont and chatFrame) then`. `self:SetUpBlizzardChatFrame` can return nil if the chatframe has been processed before.